dockerfiles/ubuntu/analytics/dashboard/Dockerfile

```diff
@@ -16,14 +16,84 @@
 #
 # ------------------------------------------------------------------------
 
-# set to product base image
-FROM wso2ei-analytics-base:6.4.0
+# set base Docker image to AdoptOpenJDK Ubuntu Docker image
+FROM adoptopenjdk/openjdk8:jdk8u192-b12
```

Comment:
@Vathsan we are using an outdated AdoptOpenJDK version here. Let's use the latest given [here](https://hub.docker.com/layers/adoptopenjdk/library/adoptopenjdk/8u282-b08-jdk-hotspot-focal/images/sha256-9e0eba11416fbb7ca70e0ebb6420f547ce049aff93990cf6eda54a47696df915?context=explore).

Reply:
Updated the AdoptOpenJDK version to **adoptopenjdk:8u282-b08-jdk-hotspot-focal** in all ubuntu base images